website-next/components/header.js

```diff
@@ -8,9 +8,34 @@ export function Header() {
       <meta name='viewport' content='width=device-width,initial-scale=1,shrink-to-fit=no' />
 
       {/* preload fonts */}
-      <link rel="preload" href="/fonts/RecklessNeue-Book.woff2" crossorigin="anonymous" as="font" type="font/woff2" />
-      <link rel="preconnect" href="https://fonts.gstatic.com"/>
-      <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet" />
+      <link
+        rel='preload'
+        href='/fonts/RecklessNeue-Book.woff2'
+        crossorigin='anonymous'
+        as='font'
+        type='font/woff2'
+      />
+      <link rel='preconnect' href='https://fonts.gstatic.com' />
+      <link href='https://fonts.googleapis.com/css2?family=Inter&display=swap' rel='stylesheet' />
+
+      {/* Google Analytics */}
```

Comment:
The below is from the sample next.js integration https://github.com/vercel/next.js/blob/canary/examples/with-google-analytics/pages/_document.js